pkg/authn/k8schain/go.mod

```diff
@@ -3,15 +3,27 @@ module github.com/google/go-containerregistry/pkg/authn/k8schain
 go 1.14
 
 require (
+	github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96 // indirect
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
```

Comment:
🤔

Reply:
yeah let me re-run the commands

Reply:
https://github.com/kubernetes/apimachinery/blob/v0.21.1/go.mod#L21

fsnotify is from gomega which k8s uses

Reply:
it's not obvious to me why the go tool chain made it show up as `//indirect` here

Reply:
`go mod tidy` FTW

Reply:
fsnotify remained - I deleted all `//indirect` dependencies and ran `go mod tidy` again